index.d.ts

```diff
@@ -0,0 +1,49 @@
+import build = require('./lib/build');
```

Comment:
This file defines the methods and properties of module-level export, ie, everything that will be available at `cf.[NAME]` after doing `const cf = require('@mapbox/cloudfriend')`.

I am not fully sure why these could not be inferred by `tsc`. Perhaps there is a way to make that happen but I could not figure it out, so explicitly defining everything in this `index.d.ts` is how I had to do it.